server/logging.go

```diff
@@ -0,0 +1,74 @@
+package server
```

Comment:
I wonder if we should add tests for this file.

Reply:
Sure!